int64 num_1
int64 num_2
---
int64 sum